server/src/main/java/au/org/aodn/ogcapi/server/core/model/SummariesModel.java

```diff
@@ -0,0 +1,11 @@
+package au.org.aodn.ogcapi.server.core.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class SummariesModel {
```

Comment:
same as `StacCollectionModel` class, have to manually sync here, if we could do stuff like a package/dependency to publish the indexer , that will reduce a lot of overhead.

Reply:
this is just my comment

Reply:
Yes but need to wait till CodeArtifact is done and your compilation prefer to generate a separate jar so that ogcapi do not needs to include everything of the indexer. 

https://github.com/aodn/backlog/issues/5235

Reply:
thank you, clear